src/tcgwars/logic/impl/gen4/Unleashed.groovy

```diff
@@ -205,12 +205,13 @@ public enum Unleashed implements LogicCardInfo {
             energyCost P
             onAttack {
               int max = self.cards.energyCount(C)
+              def tar = opp.all.findAll{ it.evolution }
```

Comment:
How do you select the second Pokémon to devolve if you move this out of the loop?